tests/Geocoder/Tests/Provider/GoogleMapsTest.php

```diff
@@ -196,60 +196,61 @@ public function testGeocodeBoundsWithRealAddressForNonRooftopLocation()
     public function testGeocodeWithRealAddressReturnsMultipleResults()
     {
         $provider = new GoogleMaps($this->getAdapter());
-        $results  = $provider->geocode('Paris');
+        $results  = $provider->geocode('i');
```

Comment:
That is very strange... But the issue is at the geocoder service. Let's do as you do an remove 'Paris'.